### ISSUE TITLE
Content language not recognized (IU Accessibility Mandate)

### ISSUE THREAD

**Issue by mdalmau** [IUBLibTech/newton_chymistry#142]
30,517 occurrences
This one is scary because it's a TEI fix. The "lang" attribute values are not standardized according to [2-letter code ISO standards](https://www.loc.gov/standards/iso639-2/php/code_list.php). 

--------------------------
IU has a mandate that all web sites published on IU servers must meet WCAG I and II Accessibility requirements. If the requirements are NOT met, sites will be taken down. To learn more about this initiative, visit the [Digital Strategy Initiative](https://ucm.iu.edu/digital-strategy/index.html) web site.

- [ ] Ask Bill who should be added to the SiteImprove tool

**Comment by mdalmau** [comment]
@aewingate fixed this issue! But @aewingate, did you re-index production too?

**Comment by aewingate** [comment]
@mdalmau: I don't know how to re-index production, so no! If you show me how, I can do that.
